presto-docs/src/main/sphinx/release/release-328.rst

```diff
@@ -0,0 +1,60 @@
+========================
+Release 328 (6 Jan 2020)
+========================
+
+General Changes
+---------------
+
+* Fix correctness issue for certain correlated JOIN queries when the subquery on the
+  right produces no rows. (:issue:`1969`)
+* Fix incorrect handling of multi-byte characters when pattern is empty. (:issue:`2313`)
```

Comment:
"in regular expressions"? in re2j or joni? @sopel39

Reply:
We should say
```
* Fix incorrect handling of multi-byte characters for :doc:`/functions/regexp` ...